cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetricsTable.java

```diff
@@ -41,6 +41,11 @@
   void put(SortedMap<byte[], ? extends SortedMap<byte[], Long>> updates);
 
   /**
+   * Write multiple rows, each with multiple individual columns of byte array type to write.
+   */
+  void putBytes(SortedMap<byte[], ? extends SortedMap<byte[], byte[]>> updates);
```

Comment:
Should I create a separate table class for `MetaMetricsTable` or just add better documentation for this method?